content/layouts/page-react.html

```diff
@@ -8,7 +8,7 @@
           <div style="padding: 2em 0;">
           <h3>{{ maintenance_line1 }}</h3>
           <h4>{{ maintenance_line2 }}</h4>
-          <a href="/"><button>Go back to Vets.gov</button></a>
+          <a href="/"><button>Go back to the Home page</button></a>
```

Comment:
@crwallace - LGTM, except "Home" does not need to be capitalized. Thanks for checking!